### ISSUE TITLE
requirements for input data?

### ISSUE THREAD

**Issue by cfandel** [cgre-aachen/gempy#99]
Hi!

I have a bunch of strike & dip points, many of which happen to be at the interface between two units. So my data points qualify as both orientations and interfaces in GemPy terminology. 
GemPy requires at least two interface points per unit and at least one orientation point per unit right? 
Can interface points and orientation points be in the same location? i.e., can I have exactly the same points in both the interface csv input file and the orientation csv input file? 
I tried this, and didn't get a clearly-related error, but the resulting model only has one layer and can't plot in 3D - the resulting error message is: 

`Potential field of the surface is outside the block. Probably is due to float errors`

Can't tell if this is related to the overlapping data points or not, and if not, where to start looking to fix. The precision of the data points doesn't extend into decimals (it's at least +- 10m in all directions), so it's odd that float errors would matter.

Thanks!

**Comment by Leguark** [comment]
Hi Chloe!
Mmm can you push the notebook to github so I can take a fast look? Github render the notebooks statically so the more plots and prints you have before commit the better

**Comment by cfandel** [comment]
Still getting used to GitHub so not sure I did it right but I think you can see it here:
[https://github.com/cfandel/gempy/blob/master/notebooks/prototype_notebooks/Chloe_Gottesacker/Gottesacker_geology_v3.ipynb](url)

**Comment by Leguark** [comment]
There you have a bunch of formations. Do you have at least two interfaces for each?

**Comment by cfandel** [comment]
Yep - 2 interfaces each and 1 orientation each at minimum, but in several cases the orientation and the interface points are at the same xyz coordinates

**Comment by cfandel** [comment]
Data here:
https://github.com/cfandel/gempy/blob/master/notebooks/prototype_notebooks/Chloe_Gottesacker/GK_orientations.csv
https://github.com/cfandel/gempy/blob/master/notebooks/prototype_notebooks/Chloe_Gottesacker/GK_interfaces.csv

**Comment by cfandel** [comment]
Also tried reducing the number of points to _only_ 2 interfaces and 1 orientation per formation, but had the same problem. All of the code works fine with a different dataset, the gottesacker_orientations.csv and gottesacker_interfaces.csv files. Only differences are that there is a fault in the other dataset, it doesn't have the location overlap, and I _think_ it's probably less accurate.

**Comment by SimonVirgo** [comment]
Hello Chloe, 
just had a quick look at your Notebook and I think the Problem might be due to wrong limits of the block. I think your xmin and xmax are not in the right range. 
cheers 
Simon

**Comment by cfandel** [comment]
Aha that fixed it! Thank you! I am still curious if it is generally bad practice to have the same points for both orientation and surfaces?

**Comment by Leguark** [comment]
Mmm I would say that it does not matter... with an 82.6 % confidence levels :D

**Comment by cfandel** [comment]
OK great! Thanks!
